.github/pull_request_template.md

```diff
@@ -0,0 +1,50 @@
+# Description
+Mandatory, single line description of the pull request, written as though it was an order.
+
+Then, following a single blank line, a summary of the change and which issue is fixed, including 
+- [ ] what is the problem being solved
+- [ ] why this is the best approach
```

Comment:
Nit but I'd prefer `why this approach was chosen` or `why this is a reasonable approach`.  Proving something is the **best** approach might be a tall order.

Reply:
agreed, though I guess the definition of "best" might include "is technically inferior, but faster to implement and the trad-eoffs are acceptable"

What about
```suggestion
- [ ] why this is a good approach
```